### DISCUSSION TITLE
Need Help (html/css): Img not responsive

### DISCUSSION THREAD

**Discussion by AndreMacedo3** [twbs/bootstrap#40874] [Q&A]
I have the following html code for a web page, the code is completely functional, I only have a problem with the logo, it doesn't resize or adjust its position as I decrease or increase the window, everything else on the site is completely responsive. 

https://github.com/AndreMacedo3/Testes.git

**Comment by julien-deramond** [comment]
Hey @AndreMacedo3 
I've edited your question to remove the attached zip file. Please provide the code and image without using a zip file, for security reasons :)

**Reply by AndreMacedo3** [reply]
Can you explain how i upload the code?

**Reply by julien-deramond** [reply]
You can use the Markdown code formatting: see https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code.
Or you can set up a reproducible example with [CodePen](https://codepen.io) or [StackBlitz](https://stackblitz.com/) if you've got a lot to share.

**Reply by AndreMacedo3** [reply]
Can i share like this?

https://github.com/AndreMacedo3/Testes.git

**Reply by julien-deramond** [reply]
Yes, sure. But can't access it. Your repository is maybe private.

**Reply by AndreMacedo3** [reply]
Updated to public
